projects/exercises/01/drive_in_line.cpp

```diff
@@ -0,0 +1,13 @@
+#include <STSL/RJRobot.h>
+
+using namespace std;
+
+int main()
+{
+    RJRobot robot(RobotType::REAL); //Initialize a RJ Robot that controls a real robot
+    
+    robot.setDriveMotors(127, 127); //Drive both robot motors forward at max speed
+    robot.Wait(2000ms); //Wait for 2000 milliseconds (2 seconds)
```

Comment:
Please refer to [RJRobot.h](https://github.com/RoboJackets/stsl/blob/master/include/STSL/RJRobot.h) for the current API.
Wait is not in the current API, but I can add it if we want. It was in last year's to be a cross-platform wait command. We technically don't need that this year since we're back to single platform. Of course, if we just like that syntax over, say, usleep(), I can add a simple wait() to RJRobot.